src/OpenCL.cpp

```diff
@@ -125,7 +132,7 @@ __kernel void in_transform(__global net_t *in, __global float *V,
 
         for (int i = 0; i < 4; i++) {
             for (int j = 0; j < 4; j++) {
-                V[(i*4 + j)*Cpad*Ppad + offset] = T2[i][j];
+                V[(i*4 + j)*CPpad + offset] = T2[i][j];
```

Comment:
Similarly
```
for (int i = 0, v_idx = offset; i < 4; i++) {
    for (int j = 0; j < 4; j++, v_idx += CPpad) {
        V[v_idx] = T2[i][j];
    }
}
```

Reply:
This one above is wrong: 
For each `j` you have the same index `v_idx`.

Reply:
Like I said I have something else in store for that part of code.

Reply:
Ah sorry. I missed that. I’ll try harder to read previous comments in future. Good luck with your idea!